meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -133,6 +133,7 @@ RESIN_CONFIGS[docker] ?= " \
     CONFIG_EXT4_FS_SECURITY=y \
     CONFIG_AUFS_FS=y \
     CONFIG_AUFS_XATTR=y \
+    CONFIG_KEYS=y \
```

Comment:
Disregard my last comment about not finding CONFIG_KEYS for rpi3. I wasn't looking correctly. It is there on a rpi3